Packages/nadena.dev.modular-avatar/Editor/ObjectRemoverProcessor.cs

```diff
@@ -0,0 +1,16 @@
+﻿using UnityEngine;
+
+namespace nadena.dev.modular_avatar.core.editor
+{
+    public static class ObjectRemoverProcessor
+    {
+        internal static void OnPreprocessAvatar(GameObject avatarGameObject)
+        {
+            var removers = avatarGameObject.transform.GetComponentsInChildren<ModularAvatarObjectRemover>(true);
+            foreach (var remover in removers)
+            {
+                foreach (var toRemove in remover.objectsToRemove) Object.DestroyImmediate(toRemove);
```

Comment:
This could throw a null reference exception if a MAOR is removed by a prior MAOR.